### PR TITLE
Cooking/Food Fixes

### PR DESCRIPTION
**Pull request by MarinaGryphon**
A MEAT

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm [hunk 1]
```diff
@@ -52,6 +63,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/meat/chicken
 	name = "chicken meat"
+	desc = "Rooster and spheroid torment, from NTPedia, the free encyclopedia."
```
Comment (chaoko99) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm]: :shipit:

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/reagent_containers/food/snacks/meat
-	name = "meat"
-	desc = "A slab of meat."
+	name = "\properany meat"
```
Comment (BurgerLUA) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm]: What is \properany ?

Comment (MarinaGryphon) [reply code/modules/reagents/reagent_containers/food/snacks/meat.dm:2]: it's named "any meat", and it's a proper noun

Comment (MarinaGryphon) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm:2]:
```suggestion
	name = "\proper any meat"
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -2728,17 +2707,16 @@
...
 	var/obj/item/weapon/reagent_containers/food/snacks/result
-	// bun + meatball = burger
-	if(istype(W,/obj/item/weapon/reagent_containers/food/snacks/meatball))
-		result = new /obj/item/weapon/reagent_containers/food/snacks/plainburger(src)
-
-	// bun + cutlet = hamburger
-	else if(istype(W,/obj/item/weapon/reagent_containers/food/snacks/cutlet))
-		result = new /obj/item/weapon/reagent_containers/food/snacks/hamburger(src)
+	// bun + meatball/cutlet = hamburger
+	else if(istype(W,/obj/item/weapon/reagent_containers/food/snacks/cutlet) || istype(W,/obj/item/weapon/reagent_containers/food/snacks/meatball))
```
Comment (MarinaGryphon) [inline code/modules/reagents/reagent_containers/food/snacks.dm:2711]:
```suggestion
	if(istype(W,/obj/item/weapon/reagent_containers/food/snacks/cutlet) || istype(W,/obj/item/weapon/reagent_containers/food/snacks/meatball))
```

**Pull request merged**